aidb/query/query.py

```diff
@@ -78,6 +74,35 @@ def _tables(self) -> Dict[str, Dict[str, str]]:
     return _tables
 
 
+  @cached_property
+  def column_name_to_aliases(self):
+    """
+    finds the mapping of column names and aliases present in the query.
+    for e.g. for the query:
+    SELECT t2.id as id, t2.frame as frame, t2.column5
```

Comment:
Is it possible for a query to have conflicting names if, e.g., you have a nested query?

Reply:
No, if there are two same tables and both of them have alias, there will be an issue. I will think about how to deal with that.